sw/device/exts/common/BUILD

```diff
@@ -0,0 +1,28 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
+
+cc_library(
+    name = "common",
```

Comment:
This isn't _really_ what this library is... it's currently in a bad place, but this should probably be called `flash_crt` or similar, since that's what it is.

It probably also wants to depend on the relevant `:crt_target`, I think.

Reply:
I think we should defer a refactoring on the various `crt` type targets until we have this initial support in place and can see what the link requirements are for `boot_rom`, example programs, `mask_rom` and the various `rom_ext`s.